cmd/agent/subcommands/streamep/command.go

```diff
@@ -0,0 +1,93 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package streamep
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+	"go.uber.org/fx"
+
+	"github.com/DataDog/datadog-agent/cmd/agent/command"
+	"github.com/DataDog/datadog-agent/comp/core"
+	"github.com/DataDog/datadog-agent/comp/core/config"
+	"github.com/DataDog/datadog-agent/comp/core/log"
+	"github.com/DataDog/datadog-agent/pkg/api/util"
+	pkgconfig "github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/logs/diagnostic"
+	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
+)
+
+// cliParams are the command-line arguments for this subcommand
+type cliParams struct {
+	*command.GlobalParams
+
+	filters diagnostic.Filters
+}
+
+// Commands returns a slice of subcommands for the 'agent' command.
+func Commands(globalParams *command.GlobalParams) []*cobra.Command {
+	cliParams := &cliParams{
+		GlobalParams: globalParams,
+	}
+	cmd := &cobra.Command{
+		Use:   "stream-ep",
```

Comment:
Could we rename this `stream-event-platform`. I know it's longer but `ep` can mean so many things in the context of the Agent.

Also we're not really streaming event but displaying them. What about creating a new `event-platform` command with a `show-event` subcommand ? I think `datadog-agent event-platform.show-event` is obvious for dev and customers alike.

Reply:
> Could we rename this `stream-event-platform`. I know it's longer but ep can mean so many things in the context of the Agent.

I think that makes sense, I can make that change

> Also we're not really streaming event but displaying them. What about creating a new `event-platform` command with a `show-event` subcommand ? I think `datadog-agent event-platform.show-event` is obvious for dev and customers alike.

I am not sure that I think this makes sense, the goal here is to mimic the existing `agent stream-logs` command, which functions in a similar way